src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -196,7 +196,11 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
                                 string path = src.ToString();
                                 if (!string.IsNullOrWhiteSpace(_parent.ImageFolder))
                                     path = Path.Combine(_parent.ImageFolder, path);
-                                dst = new Bitmap(path);
+
+                                using (var stream = new FileStream(path, FileMode.Open, FileAccess.Read))
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.drawing.bitmap.-ctor?view=netframework-4.8#System_Drawing_Bitmap__ctor_System_IO_Stream_

The Remarks section of Bitmap says:

> You must keep the stream open for the lifetime of the Bitmap.

Are we sure we can dispose this?
cc @safern

Reply:
I know for a fact it gets disposed the way our transfomers are written. When we move to the new row the object created in the previous row is disposed by the transform's that uses ImageLoader transformer.